.babelrc

```diff
@@ -13,7 +13,6 @@
         ],
         "transform-class-properties",
         "transform-object-rest-spread",
-        "transform-async-to-bluebird",
```

Comment:
Looking through Babel 6 docs, it looks like removing this means async passes through untransformed. Is that indeed what happens, or does it fall back to generators? (I assume not, since that was part of the [es2017 preset](https://babeljs.io/docs/en/6.26.3/babel-preset-es2017) that we haven't enabled here.)

Reply:
Indeed, untransformed is the intent. https://caniuse.com/#feat=async-functions